src/it/java/com/zotov/edu/passportofficerestservice/PersonBaseTest.java

```diff
@@ -0,0 +1,180 @@
+package com.zotov.edu.passportofficerestservice;
+
+import com.zotov.edu.passportofficerestservice.model.ErrorMessage;
+import com.zotov.edu.passportofficerestservice.model.PageResponse;
+import com.zotov.edu.passportofficerestservice.model.PersonSpecification;
+import com.zotov.edu.passportofficerestservice.repository.PersonsRepository;
+import io.restassured.common.mapper.TypeRef;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.UUID;
+
+import static com.zotov.edu.passportofficerestservice.util.DataConverter.convertToPersonEntities;
+import static com.zotov.edu.passportofficerestservice.util.DataConverter.convertToPersonEntity;
+import static com.zotov.edu.passportofficerestservice.util.RandomDataGenerator.*;
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public abstract class PersonBaseTest extends BaseTest {
+
+    @Autowired
+    PersonsRepository personsRepository;
+
+    PageResponse<PersonSpecification> generatePersonsPageData(int numberOfPersons, int size, int number) {
```

Comment:
It's worth using composition instead of inheritance
https://en.wikipedia.org/wiki/Composition_over_inheritance

Let's have a separate generator service used in tests

Reply:
New helpers classes have been implemented